opass

```diff
@@ -365,7 +373,10 @@ def main():
             do_git_svn_thing("rebase")
         maybe_changed = True
         try:
-            subprocess.run([pass_pgm,] + things, input=content,
+            args = [pass_pgm,] + things
+            if verbose:
+                print(args)
```

Comment:
Same here.

Reply:
Addressed in 04df5a5f17c9388a7c7fb3a79d7b6717f33cfe83